### PR TITLE
RavenDB-15254 Time Series Validations (when adding entry)

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/models/database/timeSeries/timeSeriesModel.ts [hunk 1]
```diff
@@ -45,6 +51,15 @@ class timeSeriesModel {
             ]
         });
 
+        this.tag.extend({
+            validation: [
+                {
+                    validator: () => !this.tag() || this.tag().length <= 255,
+                    message: "Number of characters cannot exceed 255."
```
Comment (ml054) [inline src/Raven.Studio/typescript/models/database/timeSeries/timeSeriesModel.ts]: can we use build validation rule?

Comment (Danielle9897) [reply src/Raven.Studio/typescript/models/database/timeSeries/timeSeriesModel.ts]: done

File: src/Raven.Studio/wwwroot/App/views/database/timeSeries/editTimeSeriesEntry.html [hunk 2]
```diff
@@ -73,19 +73,37 @@ <h4 class="modal-title">Time Series Entry for <strong data-bind="text: documentI
...
+                                <small>The maximum number of possible values (32) has been reached.</small>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="form-group" data-bind="visible: values().length > 5">
+                        <label class="control-label">&nbsp;</label>
+                        <div class="bg-warning text-warning padding padding-xs flex-horizontal">
+                            <div class="flex-start"><small><i class="icon-warning"></i></small></div>
+                            <div>
+                                <small>Note: Aggregated rollup data is Not created for a Time Series if an Entry has more than 5 values.</small>
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/database/timeSeries/editTimeSeriesEntry.html]: Please avoid using `Not`. If you need to emphasize this word please use strong or em.

Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/database/timeSeries/editTimeSeriesEntry.html]: ok

**Review by ml054 — APPROVED** [review]

**Pull request merged**